src/components/common/SummaryOfficeAddresses.vue

```diff
@@ -8,9 +8,7 @@
           <div class="meta-container__inner">
             <label><strong>Delivery Address</strong></label>
             <div class="address-wrapper">
-              <delivery-address
-                :address="registeredAddress.deliveryAddress"
-              />
+              <delivery-address :address="registeredDeliveryAddress" />
```

Comment:
This is a fix for an INSIDIOUS bug that showed up in the unit tests, whereby you get an exception when accessing property `deliveryAddress` on `null` (since `registeredDeliveryAddress` is null initially). I don't know why this shows up now. Better typing somewhere? Timing? Anyway, the solution is to use getters below (which support TS optional chaining).

Reply:
haha! A++ for using the word 'INSIDIOUS' here :)